packages/aws-cdk-lib/aws-elasticloadbalancingv2/lib/shared/base-load-balancer.ts

```diff
@@ -252,16 +252,21 @@ export abstract class BaseLoadBalancer extends Resource {
     this.setAttribute('access_logs.s3.bucket', bucket.bucketName.toString());
     this.setAttribute('access_logs.s3.prefix', prefix);
 
+    const putObjectStatement = new PolicyStatement({
+      actions: ['s3:PutObject'],
+      principals: [this.resourcePolicyPrincipal()],
+      resources: [
+        bucket.arnForObjects(`${prefix ? prefix + '/' : ''}AWSLogs/${Stack.of(this).account}/*`),
+      ],
+    });
+    bucket.addToResourcePolicy(putObjectStatement);
+    if (!bucket.policy) {
+      // Imported buckets have `autoCreatePolicy` disabled
+      bucket.policy = new s3.BucketPolicy(bucket, 'Policy', { bucket });
```

Comment:
Afaik this may fail if the imported bucket already has its bucket policy configured outside of cfn because there can only be one bucket policy for each bucket. IMO we'd better leave the bucket policy as it is, at least for this PR.

Reply:
Thanks for the review.
Should we provide users some feedback about it or assume they know how to manage bucket policies?

Reply:
Given that there's been no issue related to that, I think we can assume they know it. Additional documents about that may help, but I found that this behavior is explicitly stated [here](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_s3.Bucket.html#policy).

Reply:
Ok, just wanted feedback on this.
I think that the current documentation is clear enough.